src/openforms/registrations/contrib/email/config.py

```diff
@@ -36,3 +37,13 @@ class EmailOptionsSerializer(JsonSchemaSerializerMixin, serializers.Serializer):
             "ensure that the total file upload sizes do not exceed the email size limit."
         ),
     )
+    email_subject = serializers.CharField(
+        label=_("Email subject"),
+        help_text=_(
+            "Subject of the email sent to the registration backend. You can use the expressions "
+            "'{{ form_name }}' and '{{ submission_reference }}' to include the form name and the reference "
+            "number to the submission in the subject."
+        ),
+        required=False,
+        validators=[DjangoTemplateValidator()],
```

Comment:
Note: Errors are not shown in the frontend yet (form builder). Only the error badge on the tab appears